internal/pkg/flink/app/application.go

```diff
@@ -52,6 +52,10 @@ func StartApp(client ccloudv2.GatewayClientInterface, tokenRefreshFunc func() er
 
 	stdinBefore := utils.GetStdin()
 	consoleParser := utils.GetConsoleParser()
+	if consoleParser == nil {
+		utils.OutputErr("Error: failed to initialize console parser")
```

Comment:
I wanted to follow the style of how other `pkg/flink` code handles errors, since they just log them to `WARN`. Maybe we should clean that up in another PR?

Reply:
We already log the error resulting in `consoleParser` being `nil` to WARN, but if nothing is printed out afterwards, the command will just silently exit.

Until the CLI prints out `WARN` messages by default, I think it would be nice for the user to know their `flink shell` command failed.

Reply:
> Until the CLI prints out WARN messages by default

Got it. Can you open a ticket for this in that case?